.github/workflows/detect_version_bump.yml

```diff
@@ -0,0 +1,30 @@
+name: Version Bump merged
+
+on:
+  pull_request:
+    types:
+      - "closed"
+    branches:
+      - "main"
+
+jobs:
+  trigger-release:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged &&
+        startsWith(github.head_ref, 'bump_version_to_') &&
+        startsWith(github.event.pull_request.title, 'Bump version') &&
+        contains(github.event.pull_request.title, ' → ')
```

Comment:
Curiosity question: Is there a specific reason why all three of these need to be checked? I'm wondering why it wouldn't be sufficient to just check the branch name (or possibly branch name and the PR title `startsWith` check)

Reply:
Not a requirement. I think I started on this aspect before nailing down how much the version bump script did. Since this does trigger a new release, a false positive is a bad thing.